src/Control/Effect/Sum.hs

```diff
@@ -55,7 +43,6 @@ instance {-# OVERLAPPABLE #-} Member sub sup => Member sub (sub' :+: sup) where
   prj (R g) = prj g
   prj _     = Nothing
 
-
```

Comment:
I use two lines of whitespace to visually separate distinct chunks of the module; could we leave this in please? 🙏

Reply:
Oops, gotcha.